### DISCUSSION TITLE
Is there any way to change the runtimeConfig attribute defined in nuxt.config.ts in the plugin in nuxt 3.6.5 version?

### DISCUSSION THREAD

**Discussion by tanglongyan** [nuxt/nuxt#27166] [Questions]
**Summary (generated)**:

Is there any way to change the runtimeConfig attribute defined in nuxt.config.ts in the plugin in nuxt 3.6.5 version?

<hr>

### Environment

nuxt: 3.6.5

### Reproduction

Is there any way to change the runtimeConfig attribute defined in nuxt.config.ts in the plugin in nuxt 3.6.5 version?

### Describe the bug

Is there any way to change the runtimeConfig attribute defined in nuxt.config.ts in the plugin in nuxt 3.6.5 version?

**Comment by TheAlexLichter** [comment]
What do you mean with "Is there any way to change the runtimeConfig attribute" exactly? 🤔

**Comment by TheAlexLichter** [comment]
Duplicate of https://github.com/nuxt/nuxt/discussions/27158
